services/QuillLMS/client/app/bundles/Staff/components/comprehension/configureRules/ruleGenericAttributes.tsx

```diff
@@ -97,6 +101,14 @@ const RuleGenericAttributes = ({
         options={ruleOptimalOptions}
         value={ruleOptimal}
       />
+      {ruleID && <section className="label-status-container">
+        <p id="label-status-label">Rule ID</p>
+        <p id="label-status">{ruleID}</p>
+      </section>}
+      {ruleUID && <section className="label-status-container">
```

Comment:
Is this duplication expected?

Reply:
the specs called to show both the `id` and `uid`